### PR TITLE
Update release notes- wrong classification of file type

### PR DESCRIPTION
**Pull request by tallieber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/33122

## Description
Part of the logic in update release notes command was not updated to the FileType enum logic. 

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 1]
```diff
@@ -456,35 +359,37 @@ def update_existing_rn(self, current_rn, changed_files):
...
+                    rn_parts = new_rn.split(_header_by_type + 's')
                     new_rn_part = rn_desc
                     if len(rn_parts) > 1:
-                        new_rn = rn_parts[0] + _type + 's' + new_rn_part + rn_parts[1]
+                        new_rn = rn_parts[0] + _header_by_type + 's' + new_rn_part + rn_parts[1]
                     else:
                         new_rn = ''.join(rn_parts) + new_rn_part
                 else:
-                    new_rn_part = f'\n#### {_type}{rn_desc}'
+                    new_rn_part = f'\n#### {_header_by_type}{rn_desc}'
```
[the remark this answers is not in the document]
Comment (tallieber) [reply demisto_sdk/commands/update_release_notes/update_rn.py]: I changed the logic

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 2]
```diff
@@ -357,77 +317,18 @@ def build_rn_template(self, changed_items: dict):
...
-                if not ind_fld_header:
-                    rn_string += '\n#### Indicator Fields\n'
-                    ind_fld_header = True
-                rn_string += rn_desc
+
+            header = f'\n#### {RN_HEADER_BY_FILE_TYPE[_type]}\n'
+            rn_template_as_dict[header] = rn_template_as_dict.get(header, '') + rn_desc
+
+        for key, val in rn_template_as_dict.items():
+            rn_string += key + val
```
Comment (bakatzir) [inline demisto_sdk/commands/update_release_notes/update_rn.py:325]: use f

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 3]
```diff
@@ -456,35 +357,35 @@ def update_existing_rn(self, current_rn, changed_files):
...
                 if content_name in current_rn:
                     continue
                 else:
                     self.existing_rn_changed = True
-                    rn_parts = new_rn.split(_type + 's')
+                    rn_parts = new_rn.split(_header_by_type)
                     new_rn_part = rn_desc
                     if len(rn_parts) > 1:
-                        new_rn = rn_parts[0] + _type + 's' + new_rn_part + rn_parts[1]
+                        new_rn = rn_parts[0] + _header_by_type + new_rn_part + rn_parts[1]
```
Comment (bakatzir) [inline demisto_sdk/commands/update_release_notes/update_rn.py:377]: f

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 4]
```diff
@@ -456,35 +357,35 @@ def update_existing_rn(self, current_rn, changed_files):
...
-                if _type in new_rn:
-                    rn_parts = new_rn.split(_type + 's')
+                if _header_by_type in new_rn:
+                    rn_parts = new_rn.split(_header_by_type)
                     new_rn_part = rn_desc
                     if len(rn_parts) > 1:
-                        new_rn = rn_parts[0] + _type + 's' + new_rn_part + rn_parts[1]
-                    else:
-                        new_rn = ''.join(rn_parts) + new_rn_part
+                        new_rn = rn_parts[0] + _header_by_type + new_rn_part + rn_parts[1]
```
Comment (bakatzir) [inline demisto_sdk/commands/update_release_notes/update_rn.py:386]: f

**Review by bakatzir — APPROVED** [review]
please review the latest comments

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 5]
```diff
@@ -221,41 +222,11 @@ def find_corresponding_yml(file_path):
     def identify_changed_file_type(self, file_path):
         _file_type = None
         file_name = 'N/A'
-        if 'ReleaseNotes' in file_path or 'TestPlaybooks' in file_path:
-            return file_name, _file_type
```
Comment (amshamah419) [inline demisto_sdk/commands/update_release_notes/update_rn.py:225]: This conditional is meant to handle a case where release notes were changed, or a test playbook was changed. Is this covered somewhere else?

Comment (tallieber) [reply demisto_sdk/commands/update_release_notes/update_rn.py:225]: These files are filtered before with function filter_files_by_type - there is a global set called SKIP_RELEASE_NOTES_FOR_TYPES and these type are in it. So as I see and check these files will not reach this line.

**Review by amshamah419 — APPROVED** [review]

**Review by bakatzir — CHANGES_REQUESTED** [review]
Please add a relevant changelog line.

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 # 1.2.19
 * Fixed an issue where merge id_set was not updated to work with the new entity of Packs.
 * Added a validation that the playbook's version matches the version of its sub-playbooks, scripts, and integrations.
+* Fix an issue of wrong file type classification on update-release-notes
```
Comment (bakatzir) [inline CHANGELOG.md:6]: please move this line up under the #Changlelog
1.2.19 was already released.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 # Changelog
 * Added a validation that makes sure outputs in integrations are matching the README file.
+* Fix an issue of wrong file type classification on update-release-notes
```
Comment (bakatzir) [inline CHANGELOG.md:3]:
```suggestion
* Fixed an when files were classified incorrectly when running update-release-notes.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 # Changelog
 * Added a validation that makes sure outputs in integrations are matching the README file.
+* Fixed an issue when files were classified incorrectly when running update-release-notes.
```
Comment (bakatzir) [inline CHANGELOG.md:3]:
```suggestion
* Fixed an issue when files were classified incorrectly when running *update-release-notes*.
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**